internal/console/helpers/helpers_test.go

```diff
@@ -313,3 +315,68 @@ func TestFileAnalyzer(t *testing.T) {
 		})
 	}
 }
+
+func TestPrinter(t *testing.T) {
+	type args struct {
+		content string
+		sev     string
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "test_high",
+			args: args{
+				content: "test_high_content",
+				sev:     "HIGH",
+			},
+			want: "test_high_content",
+		},
+		{
+			name: "test_medium",
+			args: args{
+				content: "test_medium_content",
+				sev:     "MEDIUM",
+			},
+			want: "test_medium_content",
+		},
+		{
+			name: "test_low",
+			args: args{
+				content: "test_low_content",
+				sev:     "LOW",
+			},
+			want: "test_low_content",
+		},
+		{
+			name: "test_info",
+			args: args{
+				content: "test_info_content",
+				sev:     "INFO",
+			},
+			want: "test_info_content",
+		},
+		{
+			name: "test_no_sev_content",
+			args: args{
+				content: "test_no_sev_content",
+				sev:     "no_sev",
+			},
+			want: "test_no_sev_content",
+		},
+	}
+
+	printer := NewPrinter(false)
+	color.Disable()
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := printer.PrintBySev(tt.args.content, tt.args.sev)
+			if !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("PrintBySev() = %v, want = %v", got, tt.want)
```

Comment:
use `test.StringifyStruct` to make the output easier to inspect in case of failure

Reply:
resolved